lib/swaps/SwapClient.ts

```diff
@@ -217,6 +219,8 @@ abstract class SwapClient extends EventEmitter {
     },
   ): Promise<void>;
 
+  public abstract listChannels(activeOnly: boolean): Promise<any>;
```

Comment:
I don't think we actually need this as a public generic method, at least for now. If you drop this you can also remove the stub implementation for the raiden class.